### PR TITLE
feat(bcs): add content_preview to DeliveryStatusView

### PR DESCRIPTION
**Pull request by RaymondHX**
## Summary

Add `content_preview` field to `DeliveryStatusView` so the frontend can display message content in the congestion queue list without needing a separate message lookup.

## Changes

- Add `content_preview` field (`Option<String>`) to `DeliveryStatusView`
- Add `with_content_preview()` constructor that extracts text from `PersistedMessage.content` and truncates to 200 chars
- Update `delivery_notifications.rs` to pass message content in WS events
- Update `delivery_control.rs` query/cancel/resolve to include content
- Update `DeliveryAdmissionView::from` to include content preview

## Motivation

Frontend congestion display (TeamClaw) needs to show message content in the queued items list. Previously `DeliveryStatusView` only contained metadata (delivery_id, message_id, status, etc.) with no message body, making it impossible for the frontend to display what message is queued.

### CODE REVIEW

File: src/bcs/crates/service-api/bcs-service-api/src/application/message_delivery.rs [hunk 1]
```diff
@@ -38,10 +41,23 @@ impl From<&PersistedMessageDelivery> for DeliveryStatusView {
...
     }
 }
 
+impl DeliveryStatusView {
+    /// 附带消息正文预览的构造函数。
+    pub fn with_content_preview(mut self, content: Option<&serde_json::Value>) -> Self {
+        self.content_preview = content.and_then(|c| {
+            c.get("text")
+                .and_then(|t| t.as_str())
+                .map(|s| if s.len() > 200 { format!("{}…", &s[..200]) } else { s.to_string() })
```
[the remark this answers is not in the document]
Comment (RaymondHX) [reply src/bcs/crates/service-api/bcs-service-api/src/application/message_delivery.rs]: Fixed in e6da45cf5a. Replaced byte-slicing with truncate_utf8() that uses is_char_boundary() to find the nearest valid UTF-8 character boundary, preventing panics on multibyte characters. Added unit tests covering ASCII, Chinese, and emoji truncation scenarios.

File: src/bcs/crates/service-api/bcs-service-api/src/application/message_delivery.rs [hunk 2]
```diff
@@ -22,6 +22,9 @@ pub struct DeliveryStatusView {
     pub state_version: u64,
     pub run_id: Option<String>,
     pub wait_reason: Option<bcs_domain::message_delivery::DeliveryWaitReason>,
+    /// 消息正文预览（截断至前 200 字符），供前端排队列表展示。
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub content_preview: Option<String>,
```
[the remark this answers is not in the document]
Comment (RaymondHX) [reply src/bcs/crates/service-api/bcs-service-api/src/application/message_delivery.rs:27]: Fixed in e6da45cf5a. Added DeliveryStatusView field documentation table to docs/message-delivery-service-api.md including the new content_preview field (type, truncation semantics, and purpose). Added 9 unit tests covering truncate_utf8 boundary conditions (ASCII, Chinese, emoji) and with_content_preview extraction/truncation/missing-field scenarios.

**Review by vzvince — APPROVED** [review]
